SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -139,9 +139,7 @@ public void shouldReturn200WhenGettingUserById() {
     }
 
     @Test
-    public void shouldReturn401WhenGettingUserByIDUnauthorised() throws URISyntaxException {
-        URI uri = new URI(baseUri + "/" + savedUser.getId());
-
+    public void shouldReturn401WhenGettingUserByIdUnauthorised() {
```

Comment:
Does this test name/assertion need to change to reflect a 403 instead of a 401?

Reply:
No this actually does throw a 401, the unit tests check the throwing of the 403 errors.